.vscode/tasks.json

```diff
@@ -35,6 +35,55 @@
 				"npm: watch-tests"
 			],
 			"problemMatcher": []
+		},
+		{
+			"label": "prompt evaluation",
```

Comment:
As it is internal label, how about naming it 'promptfoo eval' to distinguish another one

Reply:
rename to [hidden]promptfoo eval